src/main/java/com/google/api/codegen/GapicContext.java

```diff
@@ -90,4 +94,17 @@ public CollectionConfig getCollectionConfig(Interface service, String entityName
     }
     return result;
   }
+
+  /**
+   * Returns a list of simple RPC methods.
+   */
+  public List<Method> getSimpleRpcMethods(Interface service) {
```

Comment:
@garrettjonesgoogle . My thought is that methods with and without streaming will be sufficiently different that the template will want to treat them completely separately. We can implement `getStreamingMethods()` and template for each language can call it whenever it's ready.